examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -260,17 +338,23 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
         },
         timeBin : "day",
         geoLevel: "state",
-        geoIds : [37,51,24,11,10,34,42,9,44,48,35,4,40,6,20,32,8,49,12,22,28,1,13,45,5,47,21,29,54,17,18,39,19,55,26,27,31,56,41,46,16,30,53,38,25,36,50,33,23,2]
+        geoIds : [37,51,24,11,10,34,42,9,44,48,35,4,40,6,20,32,8,49,12,22,28,1,13,45,5,47,21,29,54,17,18,39,19,55,26,27,31,56,41,46,16,30,53,38,25,36,50,33,23,2],
+        maptype: 1
```

Comment:
Define maptype constant and use it. For example, var MAP_TYPE_AGGREGATION = 1;
maptype: MAP_TYPE_AGGREGATION.